jwst/saturation/saturation.py

```diff
@@ -1,4 +1,3 @@
-
 #  Module for 2d saturation
 #
```

Comment:
```suggestion
```

Reply:
looks like this is still here after you re-requested me as a reviewer.  Did you forget about it, or do you want to keep it here?  I think it should be turned into a triple-quoted docstring or removed

Reply:
Those two lines have been removed.